server/routes.go

```diff
@@ -19,4 +19,16 @@ var routes []Route = []Route{
 		"/api/",
 		controllers.Index,
 	},
+	{
+		"HealthCheck",
+		"GET",
+		"/api/healthcheck",
```

Comment:
Since strict slashes are enabled, can you renamed this path to `/healthcheck/`. Also there is no need to add `/api/`